02-introduction.md

```diff
@@ -120,9 +120,9 @@ certain additive utility shocks (*taste shocks*) in the future.
 
 ## State Transition
 
-In the dynamic models under consideration, the state $X_t$ evolves conditional on past information $\{X_{t-1}, A_{t-1}, \dots, X_0, A_0\}$. We assume that the process satisfies the **Markov property**, i.e. the future state $X_{t+1}$ only depends on the current state $X_t$ and the action $A_t$.
+In the dynamic models under consideration, the state $X_t$ evolves conditional on past states and actions $\{X_{t-1}, A_{t-1}, \dots, X_0, A_0\}$. We assume that the process satisfies the **Markov property**, i.e. the future state $X_{t+1}$ only depends on the current state $X_t$ and the action $A_t$.
```

Comment:
Good one. Although maybe "information" might be nice to keep here, I can see arguments either way. In that case, however, use the transition from last period to the current period as the example.